rpcd/playbooks/roles/elasticsearch/templates/upgrade-pre-flight-checks.py.j2

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+# Copyright 2016, Rackspace US, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import apt
+import sys
+
+cache = apt.Cache()
+es_pkg = cache['elasticsearch']
+
+es_version = es_pkg.installed.version
+
+if es_pkg.installed.version == '{{ elasticsearch_version }}':
```

Comment:
Right, like I'm saying 13.2 is when we plan on getting rid of technical debt, so this might be something to tack on for that.